tst/EnergyPlus/unit/FuelCellElectricGenerator.unit.cc

```diff
@@ -790,6 +791,9 @@ TEST_F(EnergyPlusFixture, FuelCellTest)
     ASSERT_TRUE(process_idf(idf_objects));
     EXPECT_FALSE(has_err_output());
 
+    // allocate surface level adj ratio data member
+    state->dataHeatBal->SurfWinCoeffAdjRatioIn.dimension(6, 1.0);
+    state->dataHeatBalSurf->SurfWinCoeffAdjRatioOut.dimension(6, 1.0);
     SimulationManager::ManageSimulation(*state);
```

Comment:
The added arrays will be allocated at the beginning of `ManageSimulation`, right?